spec/03-plugins/37-opentelemetry/03-propagation_spec.lua

```diff
@@ -127,6 +150,36 @@ describe("propagation tests #" .. strategy, function()
     assert.matches("00%-" .. trace_id .. "%-%x+-01", json.headers.traceparent)
   end)
 
+  it("does not propagate w3c tracing headers when header_type set to ignore", function()
```

Comment:
should we add another test to confirm it uses the default (w3c) when `header_type=ignore` and there is an incoming tracing header that is ~= `w3c`?

Reply:
I added a test for this, where a b3 header is sent and it's set to `ignore`. In this scenario I believe traceparent should be blank, based on following the code in `propagation.lua`. Lemme know if that works!